server/lib/routes.js

```diff
@@ -62,6 +62,11 @@ module.exports = function (config, i18n) {
       res.redirect(removeVersionPrefix(req.originalUrl));
     });
 
+    // handle sign in verification links
+    app.get(addVersionPrefix('/verify_sign_in'), function (req, res) {
```

Comment:
We removed the version prefix from verification emails a while back, new mails should not have them. This endpoint (whatever we decide to call it) will never have a version prefix (famous words, eh), so shouldn't need to handle one here.